bylaws.html

```diff
@@ -11,99 +11,99 @@ <h1>CouchDB Bylaws</h1>
+<p>Users can participate by talking about the project, providing feedback, and helping others. Participation can be done at the ASF or elsewhere and includes being active on the user mailing list, third-party support forums, blogs, and social media. Users who participate in this way become contributors automatically.
 
-  <h3 id="contributors">2.2. Contributors</h3>
+<h3 id="contributors">2.2. Contributors</h3>
 
 <p><strong>A contributor is someone who makes contributions to the community, project, documentation, or code.</strong>
 
-<p>There is no special requirement to become a contributor. If you have a great idea for the project, you can get to work immediately. There is no need to ask permission. Most things can be accomplished by contributors with no special privileges or status on the project. Assistance can be provided if you need access to project resources to get your work done.
+<p>There is no special requirement to become a contributor. If you have a good idea for the project, you can get to work immediately. There is no need to ask permission. Contributors can accomplish most things with no special privileges or status on the project. We will assist you if you need access to project resources to get your work done.
 
 <p>A contributor who makes sustained contributions to the project may be invited to become a committer.
 
 <h3 id="committers">2.3. Committers</h3>
 
-<p><strong>A committer is someone who is committed to the project. In return for their commitment, they are given a binding vote in certain project decisions. Committers are hence responsible for the ongoing health of the project and the community.</strong>
+<p><strong>A committer is someone who is committed to the project. In return for your commitment, we give you a binding vote in some specific types of project decision. Committers are hence responsible for the ongoing health of the project and the community.</strong>
 
-<p>We recognise commitment in many different areas. These include, but are not limited to:
+<p>We recognize commitment in many different areas. These include, but are not limited to:
 
   <ul>
-    <li>Community (community management, ticket triage, helping new users, events etc.)
-    <li>Project (blogging, marketing, design, UX, branding, etc)
-    <li>Documentation (documentation, localisation/internationalisation, etc.)
-    <li>Code (new features, bug fixing, quality assurance, release management etc.)
+    <li>Community (e.g., community management, ticket triage, helping new users, and helping with events)
+    <li>Project (e.g., blogging, marketing, design, UX, and branding)
+    <li>Documentation (e.g., documentation, localisation, and internationalisation)
```

Comment:
Here you kept the UK spelling vs. the US one :laughing: (internationalisation vs. internationalization)